src/extension/VSProject.cs

```diff
@@ -348,9 +348,13 @@ private void LoadMSBuildProject()
             MsBuildDocument = _doc;
 
             XmlElement assemblyNameElement = (XmlElement)_doc.SelectSingleNode("/msbuild:Project/msbuild:PropertyGroup/msbuild:AssemblyName", namespaceManager);
+            if (assemblyNameElement == null)
+                throw new ApplicationException("Missing required element AssemblyName");
             string assemblyName = assemblyNameElement.InnerText;
 
             XmlElement outputTypeElement = (XmlElement)_doc.SelectSingleNode("/msbuild:Project/msbuild:PropertyGroup/msbuild:OutputType", namespaceManager);
+            if (outputTypeElement == null)
+                throw new ApplicationException("Missing required element OutputType");
```

Comment:
I was attempting to follow the style that were being used for legacy MSBuild projects in the same file, which were throwing an ApplicationException with a similar message if the OutputType/AssemblyName attributes were missing. I can certainly change to use if you feel it warrants it though.